src/utils/string-utils.ts

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/prefer-default-export
+export function removeUnderScoreCharBeforeDigits(value: string) {
+  const charBeforeDigits = value.match(/_[0-9]/g) || []
+  let finalValue = value
+  if (charBeforeDigits === null) {
+    return value
+  } else {
+    charBeforeDigits.forEach((foundChar: string) => {
+      finalValue = finalValue.replaceAll(foundChar, foundChar.split('_')[1])
+    })
+
+    return finalValue
+  }
+}
```

Comment:
There is probably a way to achieve this in one 'replaceAll' with regex. Let me see...

Reply:
```
snakeCase("01 a23 b45 c67c 89de").replace(/_(\d+)/g, '$1')
// output: 01_a23_b45_c67_c89_de
```

Ref: Eloquent Javascript, https://eloquentjavascript.net/09_regexp.html#h_k0YuTOu54D

Reply:
This is what I've been looking for yesterday. Thanks so much, I will make a quick refactor

Reply:
You're welcome :)
@object-required Could you please double-check?